lrauv_ignition_plugins/test/test_spawn.cc

```diff
@@ -96,6 +97,14 @@ TEST(SpawnTest, Spawn)
   auto spawnPub = node.Advertise<lrauv_ignition_plugins::msgs::LRAUVInit>(
     "/lrauv/init");
 
+  int sleep{0};
+  int maxSleep{30};
+  for (; !spawnPub.HasConnections() && sleep < maxSleep; ++sleep)
+  {
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+  }
+  EXPECT_LT(sleep, maxSleep);
```

Comment:
If this check hasn't returned wouldn't it be better to abort the test?

Reply:
Done 
067cfd9